pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
```

Comment:
Yes. Typically IDE plugins simply locate and call the `black` executable, which then looks for configuration defined in this file. Specifying the desired length only in the Makefile means that `black` doesn't do the right thing in general. This bit me, I was depending on an automatic vim plugin rather than calling the Makefile directly.

The Makefile works the same as before, though, because `black` finds the config file when it's called.

It's nice that `make format` is there, but it would be even better if `black` did the desired thing no matter how it was called (IDE, command line, in the Makefile).

Reply:
@araffin would you accept this black config change as a separate PR?

Reply:
I will need to check if it keeps working on my machine, otherwise we can keep it there ;)